tests/functional/spec/features/supplementary_data/supplementary_data.spec.js

```diff
@@ -433,7 +457,32 @@ describe.skip("Using supplementary data", () => {
     await expect(await $(ViewSubmittedResponsePage.productReportingContent(1)).$$(summaryItems)[3].getText()).to.equal("Value of sales from other categories");
     await expect(await $(ViewSubmittedResponsePage.productReportingContent(1)).$$(summaryValues)[0].getText()).to.equal("£110.00");
     await expect(await $(ViewSubmittedResponsePage.productReportingContent(1)).$$(summaryValues)[1].getText()).to.equal("£220.00");
-    await expect(await $(ViewSubmittedResponsePage.productReportingContent(1)).$$(summaryValues)[2].getText()).to.equal("No answer provided");
+    await expect(await $(ViewSubmittedResponsePage.productReportingContent(1)).$$(summaryValues)[2].getText()).to.equal("£440.00");
     await expect(await $(ViewSubmittedResponsePage.productReportingContent(1)).$$(summaryValues)[3].getText()).to.equal("£330.00");
   });
 });
+describe.skip("Using supplementary data (editing list item bug)", () => {
```

Comment:
The description shouldn't mention the bug, it should test some feature.

Reply:
Removed now.